### DISCUSSION TITLE
Are one-sided confidence intervals taught in intro courses?

### DISCUSSION THREAD

**Discussion by cwolock** [statdivlab/rigr#47] [General]
Unlike `t.test` from `stats` package, our `ttest` does a two-sided confidence interval regardless of the null hypothesis. I assume this is intentional, since it involves some extra steps in the code. My recollection is that intro courses (at least BIOST 511) teach only two-sided intervals. I am fine keeping this the way it is - just wanted to check since it means `ttest` doesn't correspond to `t.test` in this case. Thoughts?

**Comment by adw96** [comment]
Hi Ken & Jim! (@kenrice and @jphughes9) Do you need this functionality or would you like it? I don't discuss one-sided intervals in 515/518, though I think I assume they see them in 514/517.

**Reply by adw96** [reply]
@cwolock  -- thanks for starting this conversation! I got responses from Ken and Jim by email. 

Ken: "But one-sided tests are the lego blocks from which we can build basically anything! (https://doi.org/10.1111/rssa.12496) ;) Seriously, I don’t mind. I mention one-sided tests but only very briefly, and as non-examinable material. Either way is fine."

Jim: "I’m more of a tinker-toy guy myself …. I didn’t use one-sided confidence intervals in 511"

So sounds good -- let's keep them out 👍 Thanks!

**Reply by cwolock** [reply]
Thanks for sending along! I just realized there's a typo in my original post - should say "regardless of the alternative hypothesis." That is to say, if you run a one-sided test, you still get a two-sided interval. I gather everyone understood what I meant, but just wanted to clarify. I will leave it as-is.

**Comment by jphughes9** [comment]
Charlie

Indeed, that is how I interpreted the question.

Jim
--------------
Jim Hughes
Professor of Biostatistics
University of Washington           Email: ***@***.******@***.***>
Mailstop 351617                           Phone: 206-616-2721
Seattle, WA  98195                        Fax:      206-616-2724
                                                          Cell:      206-498-2456
